src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.targets

```diff
@@ -319,15 +359,28 @@ Copyright (c) .NET Foundation. All rights reserved.
     Computes the applicable @(ResolvedRazorConfiguration) and @(ResolvedRazorExtension) items that match the project's
     configuration. 
   -->
-  <Target
-    Name="ResolveRazorConfiguration"
-    Condition="'$(RazorDefaultConfiguration)'!=''">
-
-    <ItemGroup>
+  <Target Name="ResolveRazorConfiguration">
+    <ItemGroup Condition="'$(RazorDefaultConfiguration)'!=''">
       <ResolvedRazorConfiguration Include="@(RazorConfiguration->WithMetadataValue('Identity', '$(RazorDefaultConfiguration)')->Distinct())" />
     </ItemGroup>
 
-    <FindInList List="@(RazorExtension)" ItemSpecToFind="@(ResolvedRazorConfiguration->Metadata('Extensions'))">
+    <!--
+      ResolvedRazorConfiguration should only ever have one item in it. If misconfigured, we may resolve more than one item which
+      may result in incorrect results or poorly worded build errors.
+    -->
+    <Warning Text="Unable to resolve a Razor configuration for the value '$(_RazorConfiguration)'. Available configurations '@(RazorConfiguration)'.'"
+      Code="RazorSDK1000"
```

Comment:
Use `RZ0000` format for these I think?

Reply:
.NET SDK uses this pattern for their messages - https://github.com/dotnet/sdk/blob/afae0615d5cbb16c058c5ce1fd65215d07df24d9/src/Tasks/Common/Resources/xlf/Strings.cs.xlf (although they do use all caps which I'll fixup). Makes sense to keep the two similar.